---
title: GuildedWebsocketException(string, Exception)
layout: references
section: references
tags:
  - references
  - constructor
description: "

Creates a new instance of [GuildedWebsocketException](GuildedWebsocketException 'Guilded.NET.Base.GuildedWebsocketException') with a message received.

```csharp
public GuildedWebsocketException(string message, System.Exception inner);
```"
---

## GuildedWebsocketException(string, Exception) Constructor
###### **Assembly:** `Guilded.NET.Base`<br/>**Type:** [`GuildedWebsocketException`](GuildedWebsocketException 'Guilded.NET.Base.GuildedWebsocketException')

Creates a new instance of [GuildedWebsocketException](GuildedWebsocketException 'Guilded.NET.Base.GuildedWebsocketException') with a message received.

```csharp
public GuildedWebsocketException(string message, System.Exception inner);
```
#### Parameters

<a name='Guilded.NET.Base.GuildedWebsocketException.GuildedWebsocketException(string,System.Exception).message'></a>

`message` [System.String](https://docs.microsoft.com/en-us/dotnet/api/System.String 'System.String')

The message that was received from Guilded Websocket

<a name='Guilded.NET.Base.GuildedWebsocketException.GuildedWebsocketException(string,System.Exception).inner'></a>

`inner` [System.Exception](https://docs.microsoft.com/en-us/dotnet/api/System.Exception 'System.Exception')

The inner exception of this error